lib/screen/account/import_account_page.dart

```diff
@@ -122,6 +138,102 @@ class _ImportAccountPageState extends State<ImportAccountPage> {
                       title: 'learn_why_this_is_safe...'.tr(),
                     ),
                     const SizedBox(height: 15),
+                    Text("1. ${"select_wallet_type".tr()}.",
+                        style: theme.textTheme.ppMori400Black14),
+                    const SizedBox(height: 15),
+                    Container(
+                        decoration: BoxDecoration(
+                            border:
+                                Border.all(color: theme.colorScheme.primary),
+                            borderRadius: BorderRadiusGeometry.lerp(
+                                const BorderRadius.all(Radius.circular(5)),
+                                const BorderRadius.all(Radius.circular(5)),
+                                5)),
+                        child: Padding(
```

Comment:
we can use padding inside parent's container, no need to wrap inside a Padding Widget.